pwa-devdocs/src/peregrine/reference/list/index.md

```diff
@@ -2,11 +2,11 @@
 title: List
 ---
 
-<!-- 
+<!--
 The reference doc content is generated automatically from the source code.
 To update this section, update the doc blocks in the source code
 -->
-{% include auto-generated/peregrine/src/List/list.md %}
+{% include auto-generated/peregrine/lib/List/list.md %}
```

Comment:
I also think you missed some `hooks` doc refs in other files. Just do a search `peregrine/src`.